packages/airnode-node/src/handlers/call-api.ts

```diff
@@ -1,84 +1,122 @@
 import * as adapter from '@api3/airnode-adapter';
-import { OIS, RESERVED_PARAMETERS } from '@api3/airnode-ois';
+import { RESERVED_PARAMETERS } from '@api3/airnode-ois';
+import { ethers } from 'ethers';
+import { getMasterHDNode } from '../evm';
 import { getReservedParameters } from '../adapters/http/parameters';
 import { API_CALL_TIMEOUT, API_CALL_TOTAL_TIMEOUT } from '../constants';
+import { isValidSponsorWallet } from '../evm/verification';
```

Comment:
Minor thing, but do we want to import this from `../evm` instead? Like `getMasterNode` above

Reply:
Actually I can't - because it's exported as `verification` object and I think it's not necessary to use it as `verification.isValidSponsorWallet` for that single usage.